lms/djangoapps/courseware/module_render.py

```diff
@@ -171,7 +136,9 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
             return None
 
         # Check to see if the course is gated on required content (such as an Entrance Exam)
-        required_content = _get_required_content(course, request.user)
+        required_content = []
+        if settings.FEATURES.get('MILESTONES_APP', False):
```

Comment:
Why was a check of settings.FEATURES.get('MILESTONES_APP', False) introduced? A portion of get_required_content also checks settings.FEATURES.get('MILESTONES_APP', False).